tests/subsys/bluetooth/mesh/silvair_enocean_model/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  bluetooth.mesh.silvair_enocean_model:
+    platform_allow: native_posix qemu_cortex_m3
+    tags: bluetooth mesh models
```

Comment:
This now needs an `integration_platforms` entry, as explained in https://github.com/nrfconnect/sdk-nrf/pull/4444.
Should be `native_posix` here (as a list!) and the platforms in `platform_allow` in the sample
Same for the sample.